packages/forma-36-react-components/src/components/Card/EntryCard/EntryCard.tsx

```diff
@@ -34,6 +35,10 @@ export type EntryCardPropTypes = {
    * The publish status of the entry
    */
   status?: EntryCardStatus;
+  /**
+   * The entry has a scheduled action
+   */
+  isScheduled?: boolean;
```

Comment:
@mohamedturki I think it might be better to have a more generic `statusIcon` prop which accepts a string representing the desired icon.

Reply:
Yeah I like the idea. I will change it 👍